### PR TITLE
Add way to specify the force QP penalty weights for slack variables

### PR DESCRIPTION
**Pull request by jviereck**

### CODE REVIEW

File: src/dynamic_graph/centroidal_force_qp_controller.cpp [hunk 1]
```diff
@@ -47,6 +47,13 @@ void CentroidalForceQPController::initialize(int number_endeffectors,
         number_endeffectors, friction_coeff, qp_penalty_lin, qp_penalty_ang);
 }
 
+void CentroidalForceQPController::set_slack_weights(dynamicgraph::Vector& weights)
```
Comment (MaximilienNaveau) [inline src/dynamic_graph/centroidal_force_qp_controller.cpp:50]: I would have preferred a modification of the initialize function.
Having another setter seems unnecessary here.

File: demos/demo_robot_com_ctrl_cpp.py [hunk 2]
```diff
@@ -33,8 +33,7 @@ def demo(robot_name):
         dc = [50, 50, 50]
         kb = [100, 100, 200]
         db = [50.0, 50.0, 200.0]
-        qp_penalty_lin = 5e5
-        qp_penalty_ang = 1e6
+        qp_penalty_weights = [5e5, 5e5, 5e5, 1e6, 1e6, 1e6]
```
Comment (MaximilienNaveau) [inline demos/demo_robot_com_ctrl_cpp.py:36]: I like the 6 weights in 1 variable.

**Review by MaximilienNaveau — CHANGES_REQUESTED** [review]

**Comment by MaximilienNaveau** [comment]
In order to preserve the API you can simply overload the initialize function

File: src/dynamic_graph/centroidal_force_qp_controller.cpp (hunk 1 above)
Comment (jviereck) [reply src/dynamic_graph/centroidal_force_qp_controller.cpp:50]: Okay, I will change it.

Comment (jviereck) [reply src/dynamic_graph/centroidal_force_qp_controller.cpp]: Done.

**Comment by jviereck** [comment]
@MaximilienNaveau I've updated the PR. Please let me know if this is good to merge now.

**Pull request merged**